lib/stacker/stack.rb

```diff
@@ -200,6 +200,10 @@ def describe_change_set
     memoize :describe_change_set
 
     def pretty_change_set
+      if describe_change_set.size == 0
+        puts "Empty change set, exiting"
```

Comment:
i thought about this a bit more. it's possible that a user will want to update a template in a way that doesn't produce resource changes, but we still want to update the stack w/ the new template. i think the problem is that the `describe_change_set.map ... .max` below will make `riw` `nil` if there are no changes. when that happens, i think we can set `riw` to zero and the bug (nil exception on `riw+20`) should be fixed

Reply:
ack, updated